packages/js/api-core-tests/.gitignore

```diff
@@ -1,2 +1,4 @@
 # Collection output
 collection.json
+
+package-lock.json
```

Comment:
We should probably just ignore it at the root, since we're using PNPM throughout the entire repository, right?

Reply:
Yup makes sense. Updated. Another round please.